linters/check_columns.py

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env python
 
 import os
+import sys
 import re
 import glob
 from collections.abc import MutableSet
 
-MESA_DIR = "../"
+try:
+    MESA_DIR = os.environ["MESA_DIR"]
+except KeyError:
+    MESA_DIR = "../"
```

Comment:
If you like, you can use either of these variants of Python's [`dict.get` method](https://docs.python.org/3/library/stdtypes.html#dict.get), which has an optional return value if the key isn't in the dict. The generic version with a dict would be
```suggestion
MESA_DIR = os.environ.get("MESA_DIR", "../")
```
The `os` module also has its own similar function [`os.getenv`](https://docs.python.org/3/library/os.html#os.getenv), 
```suggestion
MESA_DIR = os.getenv("MESA_DIR", "../")
```
though I have no preference and don't know any reason to prefer one over the either.

Reply:
Nice! This makes it cleaner :)